source/_integrations/zwave_js.markdown

```diff
@@ -56,6 +56,7 @@ Some devices can be auto-discovered, which can simplify the Z-Wave JS setup proc
 | Aeotec Z-Stick Gen5+ | 0658:0200 | https://aeotec.com/z-wave-usb-stick/ |
 | Nortek HUSBZB-1 | 10C4:8A2A | https://www.nortekcontrol.com/products/2gig/husbzb-1-gocontrol-quickstick-combo/ |
 | Zooz ZST10 | 10C4:EA60 | https://www.getzooz.com/zooz-zst10-s2-stick.html |
+| Z-WaveMe UZB | 0658:0200 | https://z-wave.me/products/uzb/ |
```

Comment:
Please don't use bare links in documentation, instead link a text.
These render really badly, especially on smaller screens.